content/docs/gto/command-reference/annotate.md

```diff
@@ -0,0 +1,85 @@
+# annotate
+
+Create or update metainformation about an artifact.
```

Comment:
Why metainfo instead of metadata? We have both terms here and there in docs. Is there a difference? Should we be consistent (pick one) ?

Reply:
```suggestion
# annotate

Create or update metadata about an artifact.
```
The only difference was that `metadata` is used in MLEM, and it's not user-editable. Here it's user-editable. Np to use `metadata` everywhere.

Reply:
Sounds good.